documentation/todo-app/src/App.js

```diff
@@ -1,19 +1,37 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
-
+import Todo from './Todo';
 class App extends Component {
+  
   render() {
+    let tasks = [
+      {
+        task: " Get out of bed",
+        date: " Wed Sep 13 2017",
```

Comment:
Interesting choice to have both `date` and `deadline`. Do you mean a creation timestamp, and the due date for the task to be completed? You might want to update the names to make this more clear.